arroyo/processing/processor.py

```diff
@@ -17,13 +17,48 @@
 
 logger = logging.getLogger(__name__)
 
-LOG_THRESHOLD_TIME = 10  # In seconds
+LOG_THRESHOLD_TIME = 5  # In seconds
```

Comment:
The comment below in the implementation says 5 seconds anyway so just bringing this in line with that.